docs/guides/dynamo_deploy.md

```diff
@@ -101,7 +119,7 @@ Follow these steps to containerize and deploy your inference pipeline:
 1. Build and containerize the pipeline:
 ```bash
 # Navigate to example directory
-cd examples/hello_world
+cd $PROJECT_ROOT/examples/hello_world
 
 # Set runtime image name
 export DYNAMO_IMAGE=<dynamo_runtime_image_name>
```

Comment:
are you able to give guidance on what this image should be, as it's used as a base image during `dynamo build`. There doesn't seem to be a public image we can use for this, nor does this documentation state explicitly how/where to build it. Is it built from the Dockerfile at `container/Dockerfile.none`? Thanks!

Reply:
Added instructions here! https://github.com/ai-dynamo/dynamo/pull/498/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R53